pkgs/development/libraries/platform-folders/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub, cmake, static ? stdenv.hostPlatform.isStatic }:
```

Comment:
```suggestion
{ lib, stdenv, fetchFromGitHub, cmake }:
```

Reply:
I copied this pattern from another package, but I've now removed it since it's not that useful and probably an anti-pattern